coda-prototype/editorUI.js

```diff
@@ -400,17 +445,28 @@ var codeEditorManager =  {
             wordTextarea.find(".tag").css({'background-color': color});
             codeObj.addWords(event.item);
             regexField.val(regexMatcher.generateOrRegex(codeObj["words"]));
-
+            // update the activity stack
+            storage.saveActivity({
+                "category": "SCHEME",
+                "message": "Added word " + event.item + " to code " + codeObj.id + " in scheme " + tempScheme.id,
```

Comment:
In general I'd consider making these messages more machine parse able. Something like the psuedo-code  
```"Added word to code: " + {"word" : event.item, "code": codeObj.id}.toJSON()```